src/main/java/com/google/devtools/build/lib/rules/android/AarImport.java

```diff
@@ -292,7 +315,7 @@ private static SpawnAction createAarEmbeddedProguardExtractorActions(
             CustomCommandLine.builder()
                 .addExecPath("--input_aar", aar)
                 .addExecPath("--output_proguard_file", proguardSpecArtifact)
-                .build())
+                .build(), isPersistentAarExtractor ? WORKERS_FORCED_PARAM_FILE_INFO : null)
```

Comment:
I think putting this ternary statement in a separate statement will improve readability.